xds/src/main/java/io/grpc/xds/LoadBalancerConfigFactory.java

```diff
@@ -180,6 +184,18 @@ class LoadBalancerConfigFactory {
     return ImmutableMap.of(ROUND_ROBIN_FIELD_NAME, ImmutableMap.of());
   }
 
+  /**
+   * Builds a service config JSON object for the pick_first load balancer config based on the
+   * given config values.
+   */
+  private static ImmutableMap<String, ?> buildPickFirstConfig(Boolean shuffleAddressList) {
```

Comment:
why wrap it to Boolean again? it was bool from proto.

Reply:
Good catch. I was copying the approach used with other fields to have null semantics, but that does not apply with `bool` proto fields. Changed it now, take a look.